docs/OAUTH_PROXY.md

```
@@ -0,0 +1,328 @@
+# OAuth Proxy
+
+> Bridge traditional OAuth providers to work seamlessly with MCP's authentication flow.
+
+export const VersionBadge = ({version}) => {
+  return <code className="version-badge-container">
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: code

(MD033, no-inline-html)